pytest_allure_dsl.py

```diff
@@ -45,13 +45,13 @@ def allure_dsl(request):
 @pytest.fixture('function', autouse=True)
 def __allure_dls_pre_post_actions__(request, allure_dsl):
     if request.config.option.allure_dsl:
-        MASTER_HELPER.description(allure_dsl.description)
+        allure_dsl.setup()
```

Comment:
May be context manager would be more suitable instead of .setup() and .teardown() ?

Reply:
Done